include/cista/serialization.h

```diff
@@ -8,7 +8,7 @@
 #include <vector>
 
 #include "cista/aligned_alloc.h"
-#include "cista/containers.h"
+#include "cista/containers/bitset.h"
```

Comment:
Since serialization.h contains functions accessing details of almost all containers from `containers.h` it makes sense to include `containers.h` here.

Reply:
What was the reason for this change? Why does it work with including only bitset? Why is bitset required and the others not?

Reply:
A tool indicated the inclusion of containers.h is not necessary. Bitset is used at line 315. Other containers, eg. array, I guess are already included using other headers. I guess the intention is important, if the idea was to include all containers or to include minimal headers.

Reply:
Probably the tool was run on the test cases of cista. But since other client code might include other headers, I would rather try to not break client code and stick with the bigger `containers.h` include.

Reply:
Used containers.h